elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```diff
@@ -0,0 +1,322 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: jump;
+
+$variables: (
+  Padding: calc(#{pfe-var(container-spacer)} * 2),
+  BackgroundColor: pfe-color(surface--base),
+);
+
+:host {
+  //Default styles
+  // font-family: 'Red Hat Text', sans-serif;
+  // @include pfe-set-broadcast-theme(light);
+  // --theme: light;
+
+  // Example local variable, using BEM format
+  // --pfe-jump-links--Property: value;
+  font-weight: 400;
+  display: block;
+  position: sticky;
+  top: calc(#{pfe-var(container-spacer)} * 4);
+  padding: calc(#{pfe-var(container-spacer)} * 2);
+  --pfe-jump-links--BackgroundColor: BackgroundColor;
+}
+
+::slotted([slot="heading"]) {
+  margin: 0 0 1rem 0 !important;
+  margin-bottom: calc(#{pfe-var(container-spacer)} * 1);
+  font-size: 14px;
+  font-weight: 400;
+  text-transform: uppercase;
+}
+
+ul{
+  &.pfe-jump-links-nav {
+    &[horizontal] {
+
+    }
+    
+    padding: 1rem 1rem 1rem 0;
+    padding: 1rem 1rem 1rem 0;
+    margin: 0;
+    list-style: none;
+  }
+  &.sub-nav {
+    margin: 0;
+    padding: 0;
+    overflow-y: hidden;
+    transition: flex 1s linear;
+    display: flex;
+    flex-direction: column;
+    flex: 0;
+  }
+}
+
+li {
+  border-left: 3px solid transparent;
+  display: block;
+  text-decoration: none;
+}
+
+li.expand {
+  ul.sub-nav {
+    flex: 1;
+  }
+  a.sub-section {
+    display: table;
+    height: auto;
+  }
+}
+
+a {
+  display: table;
+  text-decoration: none;
+  font-size: 16px;
+  padding: 8px;
+  line-height: 1.5;
+  border-left: 3px solid transparent;
+  position: relative;
+  &.sub-section{
+    display: none;
+    // height: 0;
+    margin-left: 12px;
+    font-size: 14px;
+  }
+  &::after {
+    content: "";
+    position: absolute;
+    width: calc(100% - 4px);
+    height: calc(100% - 4px);
```

Comment:
why 4px? I'm confused about what this is doing

Reply:
I actually think @kylebuch8 wrote this one. IIRC it's to keep the links from "jumping around" when focused...?